src/app/service/pet/pet.service.go

```diff
@@ -276,3 +271,51 @@ func (s *Service) ChangeView(id string, in *dto.ChangeViewPetRequest) (result *d
 		Success: res.Success,
 	}, nil
 }
+
+func (s *Service) Adopt(petId string, in *dto.AdoptByRequest) (result *dto.AdoptByResponse, err *dto.ResponseErr) {
+	ctx, cancel := context.WithTimeout(context.Background(), 5*time.Second)
+	defer cancel()
+
+	res, errRes := s.petClient.AdoptPet(ctx, &petproto.AdoptPetRequest{
+		UserId: in.UserID,
+		PetId:  in.PetID,
+	})
+
```

Comment:
We can remove this blank line (line 283) because it concerns the same thing, errRes.